integration/setup/listener.go

```diff
@@ -74,23 +74,19 @@ func listenerMongoDBURI(tb testtb.TB, hostPort, unixSocketPath, newAuthDB string
 			"tls":                   []string{"true"},
 			"tlsCertificateKeyFile": []string{filepath.Join(testutil.BuildCertsDir, "client.pem")},
 			"tlsCaFile":             []string{filepath.Join(testutil.BuildCertsDir, "rootCA-cert.pem")},
-			"authMechanism":         []string{"PLAIN"},
 		}
 		user = url.UserPassword("username", "password")
-	}
-
-	path := "/"
 
-	if newAuthDB != "" {
-		q.Set("authMechanism", "SCRAM-SHA-256")
-		path += newAuthDB
```

Comment:
This change just avoids the hustles of `wireclient.Connect()` not working due to https://github.com/FerretDB/wire/blob/main/wireclient/wireclient.go#L93